tests/support/server.rs

```diff
@@ -55,10 +56,16 @@ pub fn spawn(txns: Vec<Txn>) -> Server {
             }
 
             if let Some(dur) = txn.write_timeout {
-                let headers_end = ::std::str::from_utf8(&reply).unwrap().find("\r\n\r\n").unwrap() + 4;
+                let headers_end = unsafe { ::std::str::from_utf8_unchecked(&reply) }.find("\r\n\r\n").unwrap() + 4;
```

Comment:
I changed this to just operate over the `[u8]` instead of turning it into a `str` for the sake of convenience.